services/email.js

```diff
@@ -26,34 +28,34 @@ export default {
             });
         }
         catch (e) {
-            console.log(e);
+            console.warn(e);
             return callback({message: 'Wrong SMTP configuration', error: e});
         }
 
         connection.on('error', (err) => {
-            console.log('ERROR on SMTP Client:', err);
-            if (process.env.NODE_ENV === 'test')
-                return callback(null, {email: params.to, message:  'dummy'});//DEBUG
+            console.warn('ERROR on SMTP Client:', err);
+            if (process.env.NODE_ENV !== 'production')
+                return callback(null, {email: to, message:  'dummy'});//DEBUG
             return callback({message: 'Failed creating connection to SMTP server', error: err});
         });
 
         connection.connect((result) => {
             //Result of connected event
-            console.log('Connection established with result', result, 'and connection details (options, secureConnection, alreadySecured, authenticated)', connection.options, connection.secureConnection, connection.alreadySecured, connection.authenticated);
+            // console.log('Connection established with result', result, 'and connection details (options, secureConnection, alreadySecured, authenticated)', connection.options, connection.secureConnection, connection.alreadySecured, connection.authenticated);
 
             connection.send({
-                from: params.from,
-                to: params.to
+                from: from,
+                to: to
             },
-            'From: <' + params.from + '>\r\n' + 'To: <' + params.to + '>\r\n' + 'Subject: ' + params.subject + '\r\nDate: ' + (new Date()).toGMTString() + '\r\n\r\n' + params.message,
+            'From: <' + from + '>\r\n' + 'To: <' + to + '>\r\n' + 'Subject: ' + params.subject + '\r\nDate: ' + (new Date()).toGMTString() + '\r\n\r\n' + params.message,
```

Comment:
I only suggested that because I had something else in mind, you are right of course. However in the wiki you can see
```
C: From: "Bob Example" <bob@example.com>
C: To: Alice Example <alice@example.com>
```
That's what I meant, and I thought the library supports that type of thing. But of course since both from,to are config params, no need for this either :)